File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/VehicleMaintenancePassenger.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.VehicleMaintenancePassengerCheckEngineLight
import org.openapitools.model.VehicleMaintenancePassengerDiagnosticTroubleCodes
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * Passenger vehicle data. Null if no data is available.
 * @param checkEngineLight 
 * @param diagnosticTroubleCodes Passenger vehicle DTCs.
 */
data class VehicleMaintenancePassenger (

        @JsonProperty("checkEngineLight") val checkEngineLight: VehicleMaintenancePassengerCheckEngineLight? = null,

        @JsonProperty("diagnosticTroubleCodes") val diagnosticTroubleCodes: List<VehicleMaintenancePassengerDiagnosticTroubleCodes>? = null
) {

}

File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/VehicleMaintenancePassenger.scala

package org.openapitools.client.model


case class VehicleMaintenancePassenger (
    _checkEngineLight: Option[VehicleMaintenancePassengerCheckEngineLight],
    /* Passenger vehicle DTCs. */
    _diagnosticTroubleCodes: Option[List[VehicleMaintenancePassengerDiagnosticTroubleCodes]]
)
object VehicleMaintenancePassenger {
    def toStringBody(var_checkEngineLight: Object, var_diagnosticTroubleCodes: Object) =
        s"""
        | {
        | "checkEngineLight":$var_checkEngineLight,"diagnosticTroubleCodes":$var_diagnosticTroubleCodes
        | }
        """.stripMargin
}
